updatecli/updatecli.d/jenkins-controllers/tools-jdk17.yaml

```diff
@@ -75,14 +75,14 @@ targets:
       file: config/jenkins_infra.ci.jenkins.io.yaml
       matchpattern: '_17(.*)\.(tar.gz|zip)"'
       replacepattern: '_{{ source `escapedVersion` }}.$2"'
-#     scmid: default
+    scmid: default
 
-# actions:
-#   default:
-#     kind: github/pullrequest
-#     scmid: default
-#     title: Bump JDK17 version (Jenkins tools) on all controllers to {{ source "lastVersion" }}
-#     spec:
-#       labels:
-#         - dependencies
-#         - jdk17
+actions:
+  default:
+    kind: github/pullrequest
+    scmid: default
+    title: Bump JDK17 version (Jenkins tools) on all controllers to {{ source "lastVersion" }}
+    spec:
+      labels:
+        - dependencies
+        - jdk17
```

Comment:
Good catch to uncomment these lines!

However the manifest still generates an invalid URL: it might come from error on the current manifest but it needs to be fixed to avoid failing PRs:


````
setJDK17UrlFilenames
--------------------

**Dry Run enabled**

"config/jenkins_infra.ci.jenkins.io.yaml" updated with [dry run] content "_17.0.10_7.$2\""

```
--- config/jenkins_infra.ci.jenkins.io.yaml
+++ config/jenkins_infra.ci.jenkins.io.yaml
@@ -755,15 +755,15 @@
                   - zip:
                       label: "linux && amd64"
                       subdir: "jdk-17.0.8.1+1"
-                      url: "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.8.1+1/OpenJDK17U-jdk_x64_linux_hotspot_17.0.8.1_1.tar.gz"
+                      url: "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.8.1+1/OpenJDK17U-jdk_x64_linux_hotspot_17.0.10_7.tar.gz"
                   - zip:
                       label: "windows"
                       subdir: "jdk-17.0.8.1+1"
-                      url: "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.8.1+1/OpenJDK17U-jdk_x64_windows_hotspot_17.0.8.1_1.zip"
+                      url: "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.8.1+1/OpenJDK17U-jdk_x64_windows_hotspot_17.0.10_7.zip"
                   - zip:
                       label: "linux && arm64"
                       subdir: "jdk-17.0.8.1+1"
-                      url: "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.8.1+1/OpenJDK17U-jdk_aarch64_linux_hotspot_17.0.8.1_1.tar.gz"
+                      url: "https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.8.1+1/OpenJDK17U-jdk_aarch64_linux_hotspot_17.0.10_7.tar.gz"
             - name: "jdk21"
               properties:
               - installSource:
```

````